src/main/java/org/xbill/DNS/Message.java

```diff
@@ -364,15 +365,15 @@
 getSectionRRsets(int section) {
 	if (sections[section] == null)
 		return emptyRRsetArray;
-	List<RRset> sets = new LinkedList<>();
+	List<RRset<Record>> sets = new LinkedList<>();
```

Comment:
Is there a particular reason to use a LinkedList here instead of ArrayList?

Reply:
I don't think so. It's been like that for 17 years...